skipruntime-ts/helpers/src/external.ts

```diff
@@ -64,11 +68,13 @@ export class GenericExternalService implements ExternalService {
 }
 
 type Timeout = ReturnType<typeof setInterval>;
+type Timeouts = { [name: string]: Timeout };
 
 export class TimerResource implements ExternalResource {
-  private readonly intervals = new Map<string, { [name: string]: Timeout }>();
+  private readonly intervals: { [name: string]: Timeouts } = {};
```

Comment:
Not objecting, but FMI is this change functionally significant, or mostly just style?